src/app/service/mock-data.ts

```diff
@@ -94,17 +48,17 @@ export const UserVariable: any = [{
     {
         '_id': '4394sdfdfj',
         'variableCode': '#director_signature',
```

Comment:
removed all unused mock data. i am still using system variable from this file. once i get system variable api. i will remove  this also.